telegram/ext/_dictpersistence.py

```diff
@@ -361,6 +361,34 @@ def update_callback_data(self, data: CDCData) -> None:
         self._callback_data = (data[0], data[1].copy())
         self._callback_data_json = None
 
+    def drop_chat_data(self, chat_id: int) -> None:
+        """Will delete the specified key from the :attr:`chat_data`.
+
+        .. versionadded:: 14.0
+
+        Args:
+            chat_id (:obj:`int`): The chat id to delete from the persistence.
+        """
+        if self._chat_data is None:
+            self._chat_data = defaultdict(dict)
```

Comment:
why not just `return`?

Reply:
you mean set the attribute and return, or just return without setting the attribute? I don't really know what to do here

Reply:
Well, if `_chat_data is None`, then `chat_id` is certainly not in `chat_data`. I see no reason to build a `defaultdict` here … so just

```python
if self._chat_data is None:
    return
```